storage.go

```diff
@@ -100,3 +103,54 @@ func NewStorage(driver driver.Driver, _ ...Option) Storage {
 		driver: driver,
 	}
 }
+
+// txBuilder is the internal implementation of the Tx interface.
```

Comment:
I suggest to rename to just `tx`? Builder suffix makes me think of it as it is tx factory, i.e. something that produce transaction, not the transaction itself.